homeassistant/components/peblar/strings.json

```diff
@@ -40,7 +40,7 @@
         },
         "data_description": {
           "host": "The hostname or IP address of your Peblar EV charger on your home network.",
-          "password": "The same password as you use to log in to the Peblar EV charger' local web interface."
+          "password": "The same password as you use to log into the Peblar EV charger's local web interface."
```

Comment:
This change isn't correct it is about logging into a device (as in authentication). The proposed change now making it about logging (as in logs) into a device.